fastly/block_fastly_service_v1_settings.go

```diff
@@ -26,6 +26,8 @@ func (h *SettingsServiceAttributeHandler) Process(d *schema.ResourceData, latest
 
 	if attr, ok := d.GetOk("default_host"); ok {
```

Comment:
Would this still work if you removed the `if` statement and just set `DefaultHost` to `gofastly.String(d.Get("default_host").(string)` on the initial `opts` declaration above? Given that it defaults to `""` it might be clearer if it still works?

Reply:
Yeah, that works and clearer. Thanks for the suggestion. Applied it.